Modules/Nonunit/Review/include/itkConformalFlatteningMeshFilter.h

```diff
@@ -89,7 +89,7 @@ class ITK_TEMPLATE_EXPORT ConformalFlatteningMeshFilter : public MeshToMeshFilte
   using PointIdIterator = typename CellType::PointIdIterator;
   using CellAutoPointer = typename CellType::CellAutoPointer;
 
-  /** Select the cell that will be used as reference for the flattening.
+  /** Select the cell that will be used to define the boundary/used as reference for the flattening.
```

Comment:
Maybe reflow this whole paragraph?

Reply:
Not sure I understand the term "reflow".

Reply:
Align it to about the same column width. This is a nitpick, not a requirement for merging.

Reply:
:+1: Done in 9b02586ca9687561bd7992c4d890bed769a04b4b.